.changelog/11926.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:improvement
```

Comment:
minor: seems like a bug fix since we were printing the wrong values before

Reply:
Done. I've changed this to `bug` in the latest commit.